puzzle-utils/words/anagram.py

```diff
@@ -1,5 +1,5 @@
-from utils.dictionary import get_current_dictionary
-from utils.word import get_character_histogram, histogram_to_string
+from .utils.dictionary import get_current_dictionary
```

Comment:
Is the leading `.` really necessary?

Reply:
Yeah. Python oddity with modules here. O.w. it won't work.